backend/src/main/java/de/tobias/intestinalinspector/controller/LoginController.java

```diff
@@ -62,7 +65,7 @@ public LoginController(AuthenticationManager authenticationManager, JwtService j
         try{
             authenticationManager.authenticate(authToken);
 
-            AppUserEntity appUser = appUserDetailsService.findUser(username).orElseThrow();
+            AppUserEntity appUser = appUserService.findUser(username).orElseThrow();
```

Comment:
```suggestion
            AppUserEntity appUser = appUserService.findUser(username)
              .orElseThrow(() -> new ResponseStatusException(HttpStatus.NOT_FOUND));
```

Or throw a more specialized exception and use @ControllerAdvice for mapping exceptions to status codes.